styles/variables.css

```diff
@@ -1,5 +1,5 @@
-:where(html) {
-  color-scheme: light dark;
+@define-mixin light-theme {
```

Comment:
This is only available in [postcss-mixins](https://github.com/postcss/postcss-mixins#readme) plugin. It's non-standard but helpful for defining a set of styles I'd like to apply in multiple places.